packages/fast-usdc-deploy/src/utils/deploy-config.js

```diff
@@ -49,11 +60,17 @@ export const configurations = {
       chainPolicies: ChainPolicies.TESTNET,
       eventFilter: DepositForBurnEvent,
     },
-    chainInfo: /** @type {Record<string, CosmosChainInfo & Passable>} */ (
-      withChainCapabilities({
-        agoric: fetchedChainInfo.agoric,
-        // registering USDC-on-agoric requires registering the noble chain
-        noble: fetchedChainInfo.noble,
+    chainInfo: /** @type {Record<string, ChainHubChainInfo & Passable>} */ (
+      /** @type {unknown} */ ({
```

Comment:
This was an unexpected need to cast. `ChainHubChainInfo` was meant to support this shape.